deployments/multiple-ragapps/docker-compose.yml

```
@@ -54,19 +54,26 @@ services:
       - KEYCLOAK_ADMIN_PASSWORD=admin
     volumes:
       # Mount example realm data
-      - ./keycloak/example/:/opt/keycloak/data/
+      - ${STATE_DIR:-${PWD}/data}/keycloak/example/:/opt/keycloak/data/
     networks:
       - ragapp-network
+    depends_on:
+      - traefik
 
   manager:
     image: ${MANAGER_IMAGE:-ragapp/manager:latest}
     build: ../../src/manager
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      # Mount the ragapps data folder to the manager
+      # Note: Please don't change the container path `/app/data`
+      - ${STATE_DIR:-${PWD}/data}:/app/data
     environment:
       - BASE_URL=/manager
       - RAGAPP_IMAGE=${RAGAPP_IMAGE:-ragapp/ragapp:latest}
       - CHAT_REQUEST_LIMIT_THRESHOLD=20
+      # The path in host machine where the state is stored
+      - STATE_HOST_DIR=${STATE_DIR:-${PWD}/data}
```

maybe add state dir too?
```suggestion
      - STATE_HOST_DIR=${STATE_DIR:-${PWD}/data}
      - STATE_DIR=/app/data
```